kakmerge

```diff
@@ -175,12 +197,18 @@ cmd_merge() {
 
     kak -d -s "${kak_session}" -E "${KAKMERGE_CMDS}" || log_fatal "Unable to spawn a Kakoune server"
 
-    if [ -n "${TMUX}" ]; then
+    if [ -n "${multiplexer}" ]; then
+        if type "do_create_layout_${multiplexer}" >/dev/null; then
+            do_create_layout="do_create_layout_${multiplexer}"
+        fi
```

Comment:
```suggestion
        else
            log_fatal "Unsupported multiplexer: ${multiplexer}"
        fi
```

Reply:
Nice idea, looks good together with the check for `command` 👍